client_src/js/services/octofarm-client.service.js

```diff
@@ -65,6 +66,19 @@ export default class OctoFarmClient {
     }
   }
 
+  static validateRequiredProps(input, keys) {
+    const unsetRequiredProps = keys.filter((prop) => {
+      return !input[prop];
+    });
+    if (unsetRequiredProps.length) {
+      console.error(
+        "The following properties were empty/missing in the request",
+        unsetRequiredProps
+      );
+      throw new ApplicationError(ClientErrors.FAILED_VALIDATION_PATH);
```

Comment:
key or path here? aha or just re-using my goof?

Reply:
Re-using. I was utterly confused xD

Reply:
tut tut XD

Reply:
Added TODO and passed errors to this overrides prop.